### ISSUE TITLE
When app opened, app crash (sometimes)

### ISSUE THREAD

**Issue by djkskqyr3** [BelledonneCommunications/linphone-sdk#104]
Hi, there is crash issue on app startup.
This will occur 50% of every open app. (open 10 times, 5 crash)

Here is the crash log

[testflight_feedback (2).zip](https://github.com/BelledonneCommunications/linphone-sdk/files/5795475/testflight_feedback.2.zip)

Please help me, Thanks again

_Originally posted by @djkskqyr3 in https://github.com/BelledonneCommunications/linphone-sdk/issues/98#issuecomment-757896184_

**Comment by Viish** [comment]
Please attach SDK logs from before the crash happens + a symbolized stack trace of the crash (use debug SDK).

**Comment by djkskqyr3** [comment]
how can I use dubug SDK using pod ?

**Comment by djkskqyr3** [comment]
make all enable_gpl_third_parties=no enable_debug=yes

should I build sdk with debug mode  and use?

**Comment by djkskqyr3** [comment]
@Viish I have attached testflight crash log
There is symbolized stack trace of the crash, doesn't enough for you?
Thanks

**Comment by djkskqyr3** [comment]
@Viish I got the crash log by xcode.

The last log was **dns_service_query_record_cb : resolving _sips._tcp.pn.telesonique.ch. : no such record**

Here is log file, 

[linphone_log.zip](https://github.com/BelledonneCommunications/linphone-sdk/files/5821641/linphone_log.zip)

Thanks

**Comment by djkskqyr3** [comment]
@Viish any good news?
Thanks

**Comment by Viish** [comment]
No, I still don't have the symbolized stack trace of the crash, and without it I can't do much.

**Comment by djkskqyr3** [comment]
Thanks for your reply.
How can I get  symbolized stack with debug sdk? how can I get debug sdk ?
Please advice me

**Comment by djkskqyr3** [comment]
[testflight_feedback (2).zip](https://github.com/BelledonneCommunications/linphone-sdk/files/5795475/testflight_feedback.2.zip)

[linphone_log.zip](https://github.com/BelledonneCommunications/linphone-sdk/files/5821641/linphone_log.zip)


This is not enough?
There is crash call stack and the last log before crash

**Comment by Viish** [comment]
Also please try using a 4.5-beta version, issue might already been solved.

**Comment by djkskqyr3** [comment]
already I am using 4.5-beta but there is crash yet
That is why I created this issue
